src/server/src/main/java/io/cassandrareaper/storage/CassandraStorage.java

```diff
@@ -109,6 +109,7 @@
 public final class CassandraStorage implements IStorage, IDistributedStorage {
 
   private static final int LEAD_DURATION = 600;
+  private static final UUID SCHEMA_MIGRATION_LOCK_ID = UUIDs.startOf(1);
```

Comment:
a problem i've got with this commit is the way it intrudes a temporary fix onto this class.

i think we agree that it's a temporary fix, that such a distributed migration approach would be better as a feature to the `patka/cassandra-migration` library.
but this patch is going to become difficult to undo as it's touches a few different places and methods throughout this class.

would it be possible to encapsulate most of it into an inner class, so that it will be easy to remove when a "proper fix" comes about?